sources/models/src/modeled_types/kubernetes.rs

```diff
@@ -817,3 +817,53 @@ mod test_cpu_manager_policy {
         }
     }
 }
+
+// =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
+
+/// KubernetesDurationValue represents a string that contains a valid Kubernetes duration value.
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct KubernetesDurationValue {
+    inner: String,
+}
+
+lazy_static! {
+    pub(crate) static ref KUBERNETES_DURATION_VALUE: Regex = Regex::new(
+        r"^([0-9.]+h)?([0-9.]+m)?([0-9.]+s)?([0-9.]+ms)?$"
```

Comment:
New changes:

1. simplify regex like wiping off `( )` 
2. support `ms`
2.  set standard that we do not accept formats like  `10s10h`, `10s1m` ....

Reply:
I believe this would still allow empty input; we should probably have a `ensure!(!foo.is_empty(), error::bla)` check in the `try_from` implementation.

Reply:
As Ben mentioned, it would also allow values like `...3ms` which we wouldn't want.  I think we can handle that by fixing the decimal checks, maybe something like `r"^(([0-9]+\.)?[0-9]+h)?(([0-9]+\.)?[0-9]+m)?(([0-9]+\.)?[0-9]+s)?(([0-9]+\.)?[0-9]+ms)?$"`